gen/generate.go

```diff
@@ -326,13 +340,18 @@ func generateGenesisFiles(outDir string, protoVersion protocol.ConsensusVersion,
 		fmt.Fprintln(verboseOut, protoVersion, protoParams.MinBalance)
 	}
 
+	if rewardsBalance < protoParams.MinBalance {
+		// Needs to at least have min balance
+		rewardsBalance = protoParams.MinBalance
```

Comment:
@barnjamin @jannotti Since I felt the test coverage is lacking, I made an attempt to extend coverage.  Can you review https://github.com/barnjamin/go-algorand/pull/1?

Note - It's on @barnjamin's fork.  Requires approval from @barnjamin to run CI.

Reply:
I've just merged this in fyi

Reply:
@barnjamin Thanks for bringing in the tests.  Resolving - We can address further feedback here.